src/main/java/com/cgdp/library/validation/Validator.java

```diff
@@ -0,0 +1,40 @@
+package com.cgdp.library.validation;
+
+import static com.cgdp.library.validation.IsbnValidator.isValidIsbn;
+
+import com.cgdp.library.exceptions.ValidationException;
+import java.time.LocalDate;
+import java.util.function.Supplier;
+
+public class Validator {
+
+    public static Long required(String paramName, Long value) {
+        validate(() -> value == null, "%s must not be null", paramName);
+        return value;
+    }
+
+    public static String requiredNotBlank(String paramName, String value) {
+        validate(() -> value.isBlank(), "%s must not be blank", paramName);
+        return value;
+    }
+
+    public static LocalDate requiredPastDate(String paramName, LocalDate value) {
+        if (value != null) {
+            validate(() -> value.isAfter(LocalDate.now()),
+                  "Invalid %s. %s is greater than the current date", paramName, value);
```

Comment:
greater is not a good terminology to use here as we are talking about dates not numbers
```suggestion
                  "Invalid %s. %s is after current date, paramName, value);
```